### PR TITLE
Rs 2023.03.01

### PR DESCRIPTION
**Pull request by rschlaikjer**
This one's going to look larger than it is until 14A_3 gets merged and I rebase off it

2023-03-02-214045_1920x1140_scrot

### CODE REVIEW

File: script/Ciel/Day 14/14_01B_01_CIEL14B_1C.txt [hunk 1]
```diff
@@ -659,17 +753,18 @@ I see...
 [sha:4c2426db6b327a1be80ab73ee8c6a522ba716a14]{
 -- Page 138, Offset 38718.
 -- 先輩にとってアルクェイドは強敵であっても恐怖ではないんだ。この人にとって一番怖いものは、俺がこのまま消える事だけらしい。
-Arcueid may be a powerful enemy to Senpai, but she's not afraid of her. The thing she's most afraid of is the fact I may disappear like this.
+Arcueid may be a formidable opponent, but Senpai isn't afraid of her.
+ It appears she's actually afraid of the thought that I might disappear.
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_01_CIEL14B_1C.txt:757]: contrast on first sentence is a bit lacking.

>Senpai sees Arcueid as a formidable opponent, but not as somebody to fear.
>It seems her actual, greatest fear is the thought that...

File: script/Ciel/Day 14/14_01B_01_CIEL14B_1C.txt [hunk 2]
```diff
@@ -659,17 +753,18 @@ I see...
...
 }
 [sha:fac78a6c0eaa944efb6666d18032db08cc457b1f]{
 -- Page 139, Offset 38719.
 -- 「凄いな、先輩は。
-"You're amazing, Senpai.
+"Color me impressed, Senpai.
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_01_CIEL14B_1C.txt:762]: "Color me impressed" is too tainted as a sarcastic response for me.

File: script/Ciel/Day 14/14_01B_01_CIEL14B_1C.txt [hunk 3]
```diff
@@ -679,22 +774,25 @@ When did you get so confident that you could declare victory in this situation?"
...
 -- 大丈夫、ロアには負けないって言ったでしょう。俺が勝てないのは先輩だけです」
-It's okay. I told you that I won't lose to Roa, didn't I? The only one I can't beat is you."
+And don't worry――I told you I wouldn't lose to Roa, didn't I?
+ The only person I can't win against is you."
 }
 [sha:e97cfef119c8bdaf71be234c702c611afe2a5aa4]{
 -- Page 141, Offset 38724.
 -- ありったけの信頼をこめて、笑って送り出す。
-Mustering up all the faith that I have, I pass over a smile.
+I give her a farewell smile laden with all the trust I feel.
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_01_CIEL14B_1C.txt:789]: Wow, what an awful day to be a translator. I hate Japanese.
I feel like it needs really drastic changes, but no inspiration hit.

File: script/Ciel/Day 14/14_01B_01_CIEL14B_1C.txt [hunk 4]
```diff
@@ -709,7 +807,10 @@ Senpai nods in silence, runs toward the stairs, and stops as if suddenly remembe
 [sha:2e7a60de43208b1f3873046462c2f8017de2502d]{
 -- Page 143, Offset 38728.
 -- 「はい。……その、明日になったら街に遊びに行きましょう。乾くんと三人で、この前のやり直しをするんです」
-"Yes... Um, we should go out to the city and play tomorrow. Just as we planned with Inui-kun and the two of us."
+// Merged version of 37878/37879/37880
+"Yes.
+ Um... once everything is over, we should go enjoy ourselves in the city together.
+ We can invite Inui-kun and do a re-take of the plans we made before."
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_01_CIEL14B_1C.txt:813]: From 37878/37879/37880
>Yes.
 Um... once everything is over, we should go hang out in the city together.
We can invite Inui-kun and do a re-take of the plans we made before.

File: script/Ciel/Day 14/14_01B_01_CIEL14B_1C.txt [hunk 5]
```diff
@@ -751,45 +852,49 @@ Senpai's hand brushes against mine.
...
-―――Far off in the distance, I hear rumblings coming from the urban area that's already completely changed.
+// 地響き seems to be like subterranean/earth related, so rendering as low-pitched
+―――Far off in the distance,
+ I hear a low-pitched rumble emanating from the now unrecognizable city.
 }
 [sha:8c09898a0437f1845cb0ba8578672150173f59a0]{
 -- Page 148, Offset 38742.
 -- それは大気の軋む音、世界の悲鳴だ。
-It is the sound of the creaking atmosphere; the cries of the world.
+It is the sound of the atmosphere creaking――of the world crying out.
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_01_CIEL14B_1C.txt:889]: 世界の悲鳴 is the name of this section, so noun is best.

>It is the sound of the air grating against itself――the screech of the world.

File: script/Ciel/Day 14/14_01B_01_CIEL14B_1C.txt [hunk 6]
```diff
@@ -751,45 +852,49 @@ Senpai's hand brushes against mine.
...
 -- Page 148, Offset 38743.
 -- 天変地異そのものと言える白い吸血姫。
-A white vampire princess that is the very definition of a cataclysm.
+The vampire princess in white is the very definition of a cataclysm.
 }
 [sha:95fd45ceac324f6344ae5d34232f1218e04f996b]{
 -- Page 148, Offset 38744.
 -- それを正面から打倒する為、彼女は戦場へ赴いた。
-In order to defeat it head-on, Senpai heads toward the battlefield.
+In order to face and defeat her, Senpai rushes headlong toward the battlefield.
```
Comment (Petrikow) [inline script/Ciel/Day 14/14_01B_01_CIEL14B_1C.txt:899]: Would rather bind these together.

>Faced with the cataclysmic vampire princess in white———

>Senpai rushes headlong onto the battlefield, intent on defeating her in straight battle.

File: script/Ciel/Day 14/14_01B_01_CIEL14B_1C.txt [hunk 7]
```diff
@@ -1,58 +1,85 @@
...
 -- だが、街はもう完全に色を削がれている。
-However, the city has completely lost its color.
+The city, on the other hand, has already been completely drained of color.
 }
 [sha:5acd21bf687f4b862e97b666f51b57b440431217]{
 -- Page 88, Offset 38596.
 -- いずれこのグラウンドも校舎も、あの渦のようなものに呑み込まれてしまうのだろうか……？
-Perhaps this sports ground and this building will also end up being swallowed by that whirlpool-looking thing...?
+// Don't feel the need to repeat グラウンド/校舎 on the same page
+I wonder if that strange whirlpool-looking thing will eventually swallow this place too...
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 14/14_01B_01_CIEL14B_1C.txt:23]: Prefer current, feels like the wrong structure for a perhaps here.

File: script/Ciel/Day 14/14_01B_01_CIEL14B_1C.txt [hunk 8]
```diff
@@ -257,72 +315,94 @@ In that case... Although it's a decision that shouldn't warrant even a thought,
...
-"Hwah?!"
+// Is that the 教えて！シエル先輩 SFX?
+"%{i}Hwah%{/i}?!"
 }
 [sha:4a0b1a21eedc53af89d8096bdebec8d9ef69f1c9]{
 -- Page 107, Offset 38645.
 -- いま、右の肩胛骨を絶妙な力加減で叩かれた！　ツボを押されたみたいでちょっと気持ちいい。
-Just now, she hit me using her right shoulder blade with the perfect amount of force! It bounced off me with a boing that it honestly felt kind of good.
+She just bopped me on the right shoulder with almost divine finesse!
+ I actually feel a lot better; it's like she hit a pressure point or something.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Ciel/Day 14/14_01B_01_CIEL14B_1C.txt:325]: Ehh I think this one's ok, I think even white suburban moms have heard of acupuncture at this point

File: script/Ciel/Day 14/14_01B_01_CIEL14B_1C.txt (hunk 6 above)
Comment (rschlaikjer) [reply script/Ciel/Day 14/14_01B_01_CIEL14B_1C.txt:899]: Hmm, first one makes it sound like she's already here.

File: script/Ciel/Day 14/14_01B_01_CIEL14B_1C.txt (hunk 8 above)
[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Ciel/Day 14/14_01B_01_CIEL14B_1C.txt:325]: It "makes sense", but it doesn't convey the feeling.
It's a very ありきたり comparison for someone Japanese, but even if most people know what acupuncture is, I think its unlikely most people have experienced it.

File: script/Ciel/Day 14/14_01B_01_CIEL14B_1C.txt (hunk 6 above)
Comment (Petrikow) [reply script/Ciel/Day 14/14_01B_01_CIEL14B_1C.txt:900]: It's a metaphorical face

File: script/Ciel/Day 14/14_01B_01_CIEL14B_1C.txt (hunk 7 above)
[the remark this answers is not in the document]
Comment (Petrikow) [reply script/Ciel/Day 14/14_01B_01_CIEL14B_1C.txt:23]: Hmmmmmmmmmmm
I will admit I mostly just started getting sick of seeing "I wonder", but it still feels odd to me.

Maybe just a straight question
>Will that strange whirlpool-looking thing will eventually swallow this place too...?

File: script/Ciel/Day 14/14_01B_01_CIEL14B_1C.txt (hunk 3 above)
Comment (Petrikow) [reply script/Ciel/Day 14/14_01B_01_CIEL14B_1C.txt:790]: >I send her off with a smile, doing my utmost to show just how much faith I have in her.

Best I can do.

**Pull request merged**